specification/notificationhubs/Notificationhubs.Management/models.tsp

```diff
@@ -0,0 +1,1600 @@
+   * does not analyze it.
+   */
+  @visibility(Lifecycle.Read)
+  id?: string;
+
+  /**
+   * IP address of the Private Endpoint. This is not used by Notification Hubs.
+   */
+  @visibility(Lifecycle.Read)
+  privateIpAddress?: string;
+
+  /**
+   * Link identifier. This is a string representation of an integer that is also encoded in every IPv6 frame received by Front Door,
+   * and we use it to create implicit authorization rule that allows connection from the associated Private Endpoint.
+   */
+  @visibility(Lifecycle.Read)
+  linkIdentifier?: string;
+
+  /**
+   * Group name. Always "namespace" for Notification Hubs.
+   */
+  @visibility(Lifecycle.Read)
+  groupId?: string;
+
+  /**
+   * Member name. Always "namespace" for Notification Hubs.
+   */
+  @visibility(Lifecycle.Read)
+  memberName?: string;
+}
+
+/**
+ * Represents a connectivity information to Notification Hubs namespace. This is part of PrivateLinkService proxy that tell
+ * the Networking RP how to connect to the Notification Hubs namespace.
+ */
+model GroupConnectivityInformation {
+  /**
+   * Group id. Always set to "namespace".
+   */
+  @visibility(Lifecycle.Read)
+  groupId?: string;
+
+  /**
+   * Member name. Always set to "namespace".
+   */
+  @visibility(Lifecycle.Read)
+  memberName?: string;
+
+  /**
+   * List of customer-visible domain names that point to a Notification Hubs namespace.
+   */
+  @visibility(Lifecycle.Read)
+  customerVisibleFqdns?: string[];
+
+  /**
+   * One of the domain name from the customer-visible names; this is used internally by Private Link service to make connection to Notification Hubs
+   * namespace.
+   */
+  @visibility(Lifecycle.Read)
+  internalFqdn?: string;
+
+  /**
+   * Not used by Notification Hubs.
+   */
+  @visibility(Lifecycle.Read)
+  redirectMapId?: string;
+
+  /**
+   * ARM region for Private Link Service. We use the region that contains the connected Notification Hubs namespace.
+   */
+  @visibility(Lifecycle.Read)
+  privateLinkServiceArmRegion?: string;
+}
+
+/**
+ * A customer-visible sub-resource of Private Endpoint, which describe the connection between Private Endpoint and Notification Hubs namespace.
+ */
+model PrivateLinkServiceConnection {
+  /**
+   * Name of the Private Link Service connection.
+   */
+  name?: string;
+
+  /**
+   * List of group ids. Always contains a single element - "namespace" - for Notification Hub Namespace.
+   */
+  groupIds?: string[];
+
+  /**
+   * Request message provided by the user that created the connection. This is usually used when the connection requires manual approval.
+   */
+  requestMessage?: string;
+}
+
+model OperationListResult is Azure.Core.Page<Operation>;
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+model OperationDisplay {
```

Comment:
>OperationDisplay [](http://example.com/codeflow?start=6&length=16)

[ARMBlockingComment] why not use common types Operation definitions?

Reply:
We prefer to use common type definition for this special model, which is for the special operation `Operations_List`. But it's a breaking change. we can use the common type one if we could accept breaking changes (though I think we could accept)

Reply:
Which bit of it is breaking?